### DISCUSSION TITLE
No luck so far with flux on mac. Working config.env?

### DISCUSSION THREAD

**Discussion by ghogan42** [bghira/SimpleTuner#685] [Q&A] [answered]
I'm new to SimpleTuner but not to fine tuning as a whole.
I've done a fair amount of this for sdxl, sd2.0, sd1.5 using kohya in the past.

I'm currently trying to get SimpleTuner set up for a basic Lora/Dora training on Flux (on an M3 Max with 128gb ram) and for the life of me, I can't get a run actually started.

I've followed the FLUX.md quickstart instructions and I think it's now at the point immediately before the actual training run would start.

But... I get this:
`  File "/Users/ghogan42/AI/SimpleTuner/venv/lib/python3.12/site-packages/diffusers/models/transformers/transformer_flux.py", line 41, in rope
    scale = torch.arange(0, dim, 2, dtype=torch.float64, device=pos.device) / dim
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Cannot convert a MPS Tensor to float64 dtype as the MPS framework doesn't support float64. Please use float32 instead.`

If I add the extra lines to quantize the model then the model will go through a aquanitizing step but I ultimately get a similar(same) error.

I was wondering if anyone had config.env that worked on Mac so that I could compare to? Maybe I just don't have the right combo of all the precision settings to make it work? Or the wrong diffusers version or something.

**Comment by bghira** [comment, marked as answer]
in the quickstart it notes that mac isn't supported as a viable training platform, i just use it for development - but it looks terrible, works slowly and pytorch 2.5 nightly made things worse

**Reply by ghogan42** [reply]
Sounds good. I saw you mention that you use it for development so I thought it should run despite the warning that it was unsupported.
